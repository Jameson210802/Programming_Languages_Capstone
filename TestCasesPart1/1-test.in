PROGRAM 
		KNOCKKNOCK

;
			BEGIN

  WRITE			(

'Whoâ€™s there?')





			END








