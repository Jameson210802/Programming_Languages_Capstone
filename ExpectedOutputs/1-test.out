### FILE: 1-test.in
### TOKENS (stdout)

[1;33m===== BEGIN TOKENIZE =====[0m

1 PROGRAM
2 UNKNOWN

### PARSE (stdout)

[1;35m===== BEGIN PARSING =====[0m


### STDERR (tokens + parse)
[tokens stderr]
Lexical error near: 'KNOCKKNOCK'

[parse stderr]
Parse error: expected IDENT after PROGRAM

### EXITCODES
tokens_rc=2
parse_rc=2
